examples/webgl_clipping_planes.html

```diff
@@ -0,0 +1,94 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<title>three.js webgl - clipping planes</title>
+		<meta charset="utf-8">
+		<meta name="viewport" content="width=device-width, user-scalable=no, minimum-scale=1.0, maximum-scale=1.0">
+		<style>
+			body {
+				margin: 0px;
+				background-color: #000000;
+				overflow: hidden;
+			}
+		</style>
+	</head>
+	<body>
+
+		<script src="../build/three.js"></script>
+
+		<script>
+
+			var camera, scene, renderer;
+			var mesh, plane1, plane2, startTime;
+
+			init();
+			animate();
+
+			function init() {
+
+				camera = new THREE.PerspectiveCamera( 70, window.innerWidth / window.innerHeight, 1, 1000 );
+				camera.position.z = 400;
+
+				scene = new THREE.Scene();
+
+				var texture = new THREE.TextureLoader().load( 'textures/crate.gif' );
+
+				var geometry = new THREE.BoxBufferGeometry( 200, 200, 200 );
+				var material = new THREE.MeshBasicMaterial( { map: texture, side: THREE.DoubleSide } );
+
+				mesh = new THREE.Mesh( geometry, material );
+				scene.add( mesh );
+
+				renderer = new THREE.WebGLRenderer();
+				renderer.setPixelRatio( window.devicePixelRatio );
+				renderer.setSize( window.innerWidth, window.innerHeight );
+				document.body.appendChild( renderer.domElement );
+
+				//
+
+				var N2 = Math.SQRT1_2;
+				var N3 = Math.sqrt( 1 / 3 );
+				plane1 = new THREE.Plane( new THREE.Vector3( - N3, - N3, - N3 ), 120 );
+				plane2 = new THREE.Plane( new THREE.Vector3(   0 ,  N2 , - N2 ), 90 );
+				renderer.userClippingPlanes.push( plane1, plane2 );
```

Comment:
I'd call it renderer.clippingPlanes?  I like the simple name.  Or customClippingPlanes if we need to be more specific.